README.rst

```diff
@@ -1,3 +1,8 @@
+Archived
+========
+This repositories has been archived since it has been unmaintained for a while and us not being used in the ROS project anymore.
```

Comment:
```suggestion
This repositories has been archived since it has been unmaintained for a while and us not being used in the ROS project anymore.
```

Reply:
Thanks for the improvement.

Reply:
Is it just me or is there no diff in the change and therefore it can't be accepted?

Reply:
```suggestion
This repository has been archived due to a lack of maintainance and no longer being used in the ROS project.
```

Reply:
Sorry, looks like a copy-and-paste failure.

Reply:
Thanks for the improvement.